template/experiment/research_hub/{% if project_type == 'Basic' %}basic.py{% endif %}

```diff
@@ -0,0 +1,85 @@
+"""
+Basic Workflow
+    Single condition Variable (0-1), Single Observation Variable(0-1)
+    Theorist: LinearRegression
+    Experimentalist: Random Sampling
+    Runner: Firebase Runner (no prolific recruitment)
+"""
+
+from autora.variable import VariableCollection, Variable
+from autora.experiment_runner.firebase_prolific import firebase_runner
+from autora.experimentalist.pipeline import make_pipeline
+import numpy as np
+from sklearn.linear_model import LinearRegression
+from autora.workflow.cycle import Cycle
+
+# *** Set up variables *** #
+# independent variable is coherence (0 - 1)
+# dependent variable is accuracy (0 - 1)
+variables = VariableCollection(
+    independent_variables=[Variable(name="x", allowed_values=range(1))],
+    dependent_variables=[Variable(name="y", value_range=(-1, 1))])
+
+# *** Set up the theorist *** #
+# Here we use a linear regression as theorist, but you can use other theorists included in autora (for a list: https://autoresearch.github.io/autora/)
+# Or you can set up your own theorist
+theorist = LinearRegression()
+
+# *** Set up the experimentalist *** #
+# Here we use a random sampler as experimentalist, but you can use other experimentalists included in autora (for a list:  https://autoresearch.github.io/autora/)
+# Or you can set up your own experimentalist
+uniform_random_rng = np.random.default_rng(seed=180)
+
+
+def uniform_random_sampler():
+    return uniform_random_rng.uniform(low=0, high=1, size=3)
+
+
+experimentalist = make_pipeline([uniform_random_sampler])
+
+# *** Set up the runner *** #
+# Here fill in your own credentials
+# (https://console.firebase.google.com/)
+#   -> project -> project settings -> service accounts -> generate new private key
+
+firebase_credentials = {
```

Comment:
Can firebase be used with basic configurations? I thought that it couldn't based on the firebase question in `copier.xml`.

Reply:
This is something I'll need to ask Younes about. Copied over from existing AutoRA code. Will address this in a future PR